src/internal/_reducedStream.ts

```diff
@@ -0,0 +1,40 @@
+import { RegisteredReducer, combineReducers } from '../reducer';
+import { action$ as defaultAction$ } from '../action$';
+import { Observable, Subject } from 'rxjs';
+import { ActionStream } from '../types/helpers';
+import { withNamespace } from '../operators';
+
+export type ReducedStreamOptions = {
+  errorSubject?: Subject<any>;
+  namespace?: string;
+  action$?: ActionStream;
+};
+
+/**
+ * Creates a simple reduced state stream (without tagging, making sure to have initial state, nor replay)
+ *
+ * @param initialState The initial state of the stream
+ * @param reducers The reducers that build up the stream state
+ * @param {ReducedStreamOptions} options
+ * @returns
+ */
+export const _reducedStream = <State>(
```

Comment:
Is this a new naming convention?

I think the fact that it's in `/internal/` should be enough.

Reply:
I would personally name this file and function both `reducedStream`, then rename the import (to e.g. `reducedStreamInternal` if it clashes.

In the far past, we have discussed naming conventions for private variables and concluded that prefixing things with `_` is an anti-pattern in modern JavaScript.

@chriskr I hope you remember the same thing I do :)

Reply:
@holographic-principle  Changed 👌🏼